lld/test/wasm/function-pointer-alignmnet.s

```diff
@@ -31,207 +34,28 @@ func2:
-# CHECK-3-NEXT:   Version:         0x1
-# CHECK-3-NEXT: Sections:
-# CHECK-3-NEXT:   - Type:            TYPE
-# CHECK-3-NEXT:     Signatures:
-# CHECK-3-NEXT:       - Index:           0
-# CHECK-3-NEXT:         ParamTypes:      []
-# CHECK-3-NEXT:         ReturnTypes:     []
-# CHECK-3-NEXT:   - Type:            FUNCTION
-# CHECK-3-NEXT:     FunctionTypes:   [ 0, 0, 0 ]
-# CHECK-3-NEXT:   - Type:            TABLE
-# CHECK-3-NEXT:     Tables:
-# CHECK-3-NEXT:       - Index:           0
-# CHECK-3-NEXT:         ElemType:        FUNCREF
-# CHECK-3-NEXT:         Limits:
-# CHECK-3-NEXT:           Flags:           [ HAS_MAX ]
-# CHECK-3-NEXT:           Minimum:         0x6
-# CHECK-3-NEXT:           Maximum:         0x6
-# CHECK-3-NEXT:   - Type:            MEMORY
-# CHECK-3-NEXT:     Memories:
-# CHECK-3-NEXT:       - Minimum:         0x2
-# CHECK-3-NEXT:   - Type:            GLOBAL
-# CHECK-3-NEXT:     Globals:
-# CHECK-3-NEXT:       - Index:           0
-# CHECK-3-NEXT:         Type:            I32
-# CHECK-3-NEXT:         Mutable:         true
-# CHECK-3-NEXT:         InitExpr:
-# CHECK-3-NEXT:           Opcode:          I32_CONST
-# CHECK-3-NEXT:           Value:           66560
-# CHECK-3-NEXT:   - Type:            EXPORT
-# CHECK-3-NEXT:     Exports:
-# CHECK-3-NEXT:       - Name:            memory
-# CHECK-3-NEXT:         Kind:            MEMORY
-# CHECK-3-NEXT:         Index:           0
-# CHECK-3-NEXT:       - Name:            _start
-# CHECK-3-NEXT:         Kind:            FUNCTION
-# CHECK-3-NEXT:         Index:           0
-# CHECK-3-NEXT:   - Type:            ELEM
+# CHECK-3:        - Type:            ELEM
 # CHECK-3-NEXT:     Segments:
 # CHECK-3-NEXT:       - Offset:
 # CHECK-3-NEXT:           Opcode:          I32_CONST
 # CHECK-3-NEXT:           Value:           1
-# CHECK-3-NEXT:         Functions:       [ 1, 0, 2, 0, 0 ]
-# CHECK-3-NEXT:   - Type:            CODE
-# CHECK-3-NEXT:     Functions:
-# CHECK-3-NEXT:       - Index:           0
-# CHECK-3-NEXT:         Locals:          []
-# CHECK-3-NEXT:         Body:            4181808080001A4183808080001A0B
-# CHECK-3-NEXT:       - Index:           1
-# CHECK-3-NEXT:         Locals:          []
-# CHECK-3-NEXT:         Body:            0B
-# CHECK-3-NEXT:       - Index:           2
-# CHECK-3-NEXT:         Locals:          []
-# CHECK-3-NEXT:         Body:            0B
-# CHECK-3-NEXT:   - Type:            CUSTOM
-# CHECK-3-NEXT:     Name:            name
-# CHECK-3-NEXT:     FunctionNames:
-# CHECK-3-NEXT:       - Index:           0
-# CHECK-3-NEXT:         Name:            _start
-# CHECK-3-NEXT:       - Index:           1
-# CHECK-3-NEXT:         Name:            func1
-# CHECK-3-NEXT:       - Index:           2
-# CHECK-3-NEXT:         Name:            func2
-# CHECK-3-NEXT:     GlobalNames:
-# CHECK-3-NEXT:       - Index:           0
-# CHECK-3-NEXT:         Name:            __stack_pointer
+# CHECK-3-NEXT:         Functions:       [ 1, 0, 2, 0, 0, 3, 0, 0 ]
```

Comment:
It looks like first function is always getting address 1 here.. which seems wrong, no?